.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: objective-c
-osx_image: xcode7.3
+osx_image: xcode8
 script: 
  - xcodebuild test -project PSOperations.xcodeproj -scheme PSOperations -destination 'platform=iOS Simulator,name=iPhone 6s' -destination 'platform=tvOS Simulator,name=Apple TV 1080p'
  - xcodebuild test -project PSOperations.xcodeproj -scheme PSOperationsAppForTests -destination 'platform=iOS Simulator,name=iPhone 6s'
```

Comment:
Suggest turning off code signing. Just append  ` CODE_SIGNING_REQUIRED=NO` to the end of you `xcodebuild` command.
Similar to this commit https://github.com/Electrode-iOS/ELReachability/commit/5ef915f53065bc585ec7f3c54e93e18f7dca9f79